server/scripting/script_server.cpp

```diff
@@ -294,6 +322,9 @@ bool script_server_init()
     return false;
   }
 
+  sol::state_view sol_state(fcl_main->state);
```

Comment:
What kind of permissions does the client lua state have?
Is it reset when disconnecting?
Using the same state as for client scripts allows a broken server to mess up with it, I would prefer a different one.